workflow/create_experiment.py

```diff
@@ -79,7 +79,7 @@ def input_args():
 
     if 'skip_ci_on_hosts' in testconf:
         host = Host()
-        if host.machine.lower() in testconf.skip_ci_on_hosts.lower():
+        if host.machine.lower() in ''.join([x.lower() for x in testconf.skip_ci_on_hosts]):
```

Comment:
```suggestion
        if host.machine.lower() in [ machine.lower() for machine in testconf.skip_ci_on_hosts ]:
```

Reply:
That join is so that the use can use a "-" or not in the YAML.  If you take that out it will fail.

Reply:
Ok that works when we use a dash in the YAML.  If there is no dash it will not work but not fail.

Reply:
This is how list of things are supposed to be designated in YAML and I agree it is better to be consistent.